src/theia/sfm/pose/six_point_radial_distortion_homography.cc

```diff
@@ -0,0 +1,157 @@
+// Copyright (C) 2019 The Regents of the University of California (Regents).
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are
+// met:
+//
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//
+//     * Redistributions in binary form must reproduce the above
+//       copyright notice, this list of conditions and the following
+//       disclaimer in the documentation and/or other materials provided
+//       with the distribution.
+//
+//     * Neither the name of The Regents or University of California nor the
+//       names of its contributors may be used to endorse or promote products
+//       derived from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+//
+// Please contact the author of this library if you have any questions.
+// Author: Chris Sweeney (cmsweeney@cs.ucsb.edu)
+
+// This file was created by Steffen Urban (urbste@googlemail.com) or
+// company address (steffen.urban@zeiss.com)
+// January 2019
+
+#include "theia/sfm/pose/six_point_radial_distortion_homography.h"
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+#include <Eigen/Jacobi>
+
+namespace theia {
+
+using Eigen::Matrix;
+using Eigen::MatrixXd;
+using Eigen::Vector3d;
+using Eigen::Vector2d;
+using Vector6d = Eigen::Matrix<double, 6, 1>;
+using Array51d = Eigen::Array<double, 5, 1>;
+using Matrix68d = Eigen::Matrix<double, 6, 8>;
+using Matrix62d = Eigen::Matrix<double, 6, 2>;
+using Matrix65d = Eigen::Matrix<double, 6, 5>;
+using Eigen::Matrix3d;
+
+bool isNearZero(double val) {
```

Comment:
```suggestion
bool IsNearZero(double val) {
```

Reply:
I am using qtcreator and the Beautifier with clang to format the files. In the preferences I can choose between styles: LLVM, Google, WebKit and some others. Which one do you use?